hash/saturn.xml

```diff
@@ -5194,6 +5226,23 @@ https://segaretro.org/Delisoba_Deluxe
 		</part>
 	</software>
 
+	<!-- Demo disc bundled with Keiou Yuugekitai: Katsugeki-hen Kouryaku & Settei Shiryou-shu book -->
+	<software name="keio2jd" cloneof="keio2" supported="no">
+		<description>Keiou Yuugekitai - Okiraku Tamatebako (Japan)</description>
```

Comment:
If a demo then it needs the `demo` moniker in the parenthesis. Also if said demo is playable up to the end then it needs  `supported="partial"` (iirc game uses unemulated transparency shenanigans already).